lib/stitches/generator_files/spec/features/api_spec.rb.erb

```diff
@@ -48,6 +48,8 @@ feature "general API stuff" do
     expect(response).to have_api_error(code: "test", message: "OH NOES!")
   end
 
+<%# Remove always false if statement to enable API Key authentication %>
+<% if false %>
```

Comment:
This kind of jumps out at me but it'll work and I don't have a snappy alternative.

Reply:
I'm definitely open to changing this now or in the future. Every other "block comment" approach I tried ended up getting mangled with the other ERB.